save_dashie.py

```diff
@@ -55,8 +51,26 @@ def monitor_dashie():
 if __name__ == "__main__":
     freeze_support()
 
+    import argparse
+    import getpass
+
+    parser = argparse.ArgumentParser(description='Save dashie on /r/places!')
+    parser.add_argument('--user', help='Reddit username')
+    parser.add_argument('--passwd', help='Reddit password')
+    parser.add_argument('--url', help='WebSockets URL')
+
+    args = parser.parse_args()
+    if args.user is None:
+        args.user = raw_input("Enter Reddit Username: ")
+    if args.passwd is None:
+        args.passwd = getpass.getpass()
```

Comment:
It already shows `"Password: "` as the default prompt. Is that fine, or do you want a different prompt?